group_vars/development

```diff
@@ -3,10 +3,10 @@ mysql_root_password: devpw
 web_user: vagrant
 
 wordpress_sites:
-  example.dev:
+  example.com:
```

Comment:
However, for `group_vars/development`, I think site name should remain `.dev` because maybe you do have a live `.com` even while working on the `.dev`. You wouldn't want vagrant-hostsupdater messing with the hosts file entry for `.com`. So, in `group_vars/development`, I'd suggest the defaults be site name `example.dev` but local path `example.com`.